app/controllers/timelineable.rb

```diff
@@ -6,13 +6,13 @@ module Timelineable
   end
 
   def timeline
-    @display_date = display_date_as_time.to_date
+    @display_date = display_date_as_time
```

Comment:
Mistaking it for a `Date` value was the root of https://github.com/tablexi/nucore-open/pull/328, where the unavailability display disappeared from timelines.